src/test-run/index.js

```diff
@@ -148,6 +162,25 @@ export default class TestRun extends Session {
         return new Promise((resolve, reject) => this.pendingDriverTask = { command, resolve, reject });
     }
 
+    _addPendingWindowTask () {
+        var command = new ExecutePendingBrowserManipulationCommand();
```

Comment:
I just thought that we choose not quite correct name for this client command. It's actually prepares browser manipulation execution, while manipulation executed on server. Maybe rename it to `PrepareBrowserManipulation`? Rename executor to `prepare-browser-manipulation.js`. And client message to `ready-for-browser-manipulation`. \cc @VasilyStrelyaev